app/scripts/modules/core/src/diffs/viewChangesLink.component.ts

```diff
@@ -1,3 +1,4 @@
+import {extend} from 'lodash';
```

Comment:
I don't love what's happened to this component (and it's all my fault).  I think it needs to be more normalized across the two places it's used such that it's completely agnostic as to the data source.